components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.spec.ts

```diff
@@ -79,13 +76,7 @@ describe('PendingEditsBarComponent', () => {
         ReactiveFormsModule,
         RouterTestingModule,
         ChefPipesModule,
-        StoreModule.forRoot(
-          {
-          policies: policyEntityReducer,
-          projects: projectEntityReducer,
-          notifications: notificationEntityReducer, // not used here but needed to suppress warnings
-          clientRunsEntity: clientRunsEntityReducer // not used here but needed to suppress warnings
-        }, { runtimeChecks })
+        StoreModule.forRoot(ngrxReducers, { runtimeChecks })
```

Comment:
All the 18 other files have this change: moving from specific reducers being called out to using the master collection en masse, because the above fixes caused hundreds of warnings when running the unit tests from the command line--see the individual commit for details.

Reply:
Thanks for this cleanup! So much cleaner now ⭐️